### PR TITLE
Contact form added

### PR DESCRIPTION
**Pull request by ehs9nino**

### CODE REVIEW

**Comment by ehs9nino** [comment]
In the contact form section, I worked with my coding partner Sameh & we created a UX form in which all the widgets are designed inside the "< UL >" tag. Also, we used the flexbox and we made our UX user-friendly. We integrated & storing our  UX data in the Formspree platform. 
 The template really meets the requirements that are provided in Figma. I would like to see any good suggestions if we have mistakes, please!

File: style.css [hunk 1]
```diff
@@ -394,3 +395,118 @@ p {
...
+  border: none;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  padding: 12px;
+  width: 303px;
```
Comment (nicupop729) [inline style.css:479]: Our web page should be responsive to different types and sizes of screen devices.

- [ ] Please remove the fixed width, and set the width as wide as of the inputs; :+1:

File: style.css [hunk 2]
```diff
@@ -394,3 +395,118 @@ p {
...
+  height: 90px;
+}
+
+.form-button {
+  text-align: left;
+  margin-left: 112px;
+}
```
Comment (nicupop729) [inline style.css]: - [ ] Kindly, try to centre in the middle of the page the form button in order to match the Figma design;
Figma design:
image

Your design:
image

File: style.css [hunk 3]
```diff
@@ -394,3 +395,118 @@ p {
...
+  text-align: center;
+  color: #fff;
+  background-color: #ff6b00;
+  padding: 12px;
+}
+
+.form-button :hover {
+  background-color: #fff5e1;
+  color: #ff6b00;
+}
```
Comment (nicupop729) [inline style.css:506]: Button interactions such as hover, active (when it's pressed), disabled and enable (default state) should be styled as shown in the Figma template.
Please implement the missing states for the links/buttons in your app (at least for the pressed state / active state).
Here is a very useful link to study different states for links/buttons: [Styling links](https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Styling_links)

Figma design:
image

**Review by nicupop729 — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️

Hi @ehs9nino & @sameh080081, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### Required Changes ♻️

- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ehs9nino** [comment]
@nicupop729 thanks for identifying the errors in my mobile- portfolio. 
I tried to fix all the issues according to my understanding of the fixes you recommended, I am sure it will be more helpful if I apply these changes on my desktop versions as well.

**Review by Deepakdanger — APPROVED** [review]
Hi @ehs9nino, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ehs9nino** [comment]
Thanks, @Deepakdanger for the review!

**Pull request merged**